Frontend/ui-driver/src/Screens/ReportIssueChatScreen/Controller.purs

```diff
@@ -310,39 +322,36 @@ eval (ViewImageModelAction (ViewImageModel.BackPressed)) state = do
 ---------------------------------------------------- Record Audio Model ----------------------------------------------------
 eval (RecordAudioModelAction (RecordAudioModel.OnClickRecord push)) state = do
    continueWithCmd state { data { recordAudioState { timer = "00:00" } } }  [do
-    cond <- startAudioRecording ""
-    if cond
-    then do
-      _ <- pure $ clearTimer ""
-      _ <- removeMediaPlayer ""
-      _ <- startTimer 0 false push RecordAudioModel.TimerCallback
+    recordingStarted <- runEffectFn1 startAudioRecording ""
+    if recordingStarted then do
+      _ <- pure $ clearWaitingTimer state.props.timerId
```

Comment:
try to use `void` everywhere instead of _ <-

Reply:
changed as much I found